src/Components/Components/src/RenderTree/RenderTreeDiffBuilder.cs

```diff
@@ -23,7 +23,8 @@ enum DiffAction { Match, Insert, Delete }
         RenderBatchBuilder batchBuilder,
         int componentId,
         ArrayRange<RenderTreeFrame> oldTree,
-        ArrayRange<RenderTreeFrame> newTree)
+        ArrayRange<RenderTreeFrame> newTree,
+        Dictionary<string, int>? namedEventIndexes)
```

Comment:
```suggestion
        Dictionary<string, int>? namedEventIndices)
```

Though I feel bad for saying so. If this is not public API, feel free to ignore this point :)

Reply:
Is this a UK thing? 😄. I don't mind either way. https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/index-indexes-indices

Reply:
It's not important at all! Let's forget about this :)